tools/github/print-version-bump-info.ts

```diff
@@ -3,11 +3,17 @@ import yargs from "yargs";
 import { getCommitAndLabels } from "./github-utils";
 
 async function printInfo(octokit: Octokit, previousVersion: string, nextVersion: string) {
-  const owner = "paritytech";
+  const owners = {
+    substrate: "paritytech",
+    polkadot: "paritytech",
+    cumulus: "paritytech",
+    nimbus: "purestake",
+  };
   const prefixes = {
     substrate: "polkadot-",
     polkadot: "release-",
     cumulus: "polkadot-",
+    nimbus: "moonbeam-polkadot-",
```

Comment:
I don't know if this is relevant or not. I considered making a `polkdot-vx.y.z` branch in nimbus too for a few reasons:
1. If other projects want to use it, they have a better starting point.
2. For CI it is good to be sure nimbus works with the stock dependencies as well as moonbeam's hotfix ones.

That being said, I don't know if the team thinks that plan is worth the overhead.

Reply:
It's a good idea, when the nimbus repository will use `polkadot-vx.y.z` branchs, we will update this script, but for now this script must be based on the existing one.